source/codegen/metadata/nidcpower/functions.py

```diff
@@ -695,7 +695,7 @@
         'name': 'eventId',
         'direction': 'in',
         'type': 'ViInt32',
-        'enum': 'ExportSignal'
```

Comment:
Note: `ExportSignal` contains enum values from both `Event` and `SendSoftwareEdgeTriggerType`. But for `WaitForEventWithChannels` eventId is only using the `Event` values: https://github.com/ni/grpc-device/wiki/NI-DCPOWER-Trigger-And-Event-Functions#parameters-35